extensions/amp-story/0.1/amp-story-auto-ads.js

```diff
@@ -321,6 +321,9 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     a.href = ctaUrl;
     a.textContent = ctaText;
 
+    user().assert(['https:', 'http:'].includes(a.protocol),
```

Comment:
let's return false

Reply:
Can't you use: https://github.com/ampproject/amphtml/blob/9b101fa000b7dc911073cacdf3ccafb187c2012d/src/url.js#L268-L273

Reply:
discussed with @lannka offline. We want to discard the ad, not stop the whole component. Refactored.